<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleaningDialog</class>
 <widget class="QDialog" name="CleaningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>清洗数据</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_0">
    <attribute name="title">
     <string>加载</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>10</y>
       <width>791</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>数据</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>20</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>文件</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="rawDataPath_lineEdit">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>20</y>
           <width>501</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="rawDataPath_pushButton">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>20</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>80</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>编码</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>80</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>语种</string>
         </property>
        </widget>
        <widget class="QComboBox" name="rawDataEncoding_comboBox">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>80</y>
           <width>87</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>UTF-8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ANSI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UNICODE</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="rawDataLang_comboBox">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>80</y>
           <width>87</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>中文</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>英文</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="rawDataType_comboBox">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>80</y>
           <width>251</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>题录—知网—es5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>题录—GBT 7714-2015</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>题录—知网—NoteExpress</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>80</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>类型</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>时间</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>70</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>  从</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>70</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>  到</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="startDate_lineEdit">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>70</y>
           <width>150</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="endDate_lineEdit">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>70</y>
           <width>150</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>70</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>粒度</string>
         </property>
        </widget>
        <widget class="QComboBox" name="rawDatePeriod_comboBox">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>70</y>
           <width>87</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>1年</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1月</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1日</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2年</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3年</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>状态</string>
        </property>
        <widget class="QLabel" name="status_label">
         <property name="geometry">
          <rect>
           <x>14</x>
           <y>29</y>
           <width>761</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(170, 255, 255)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>14</x>
           <y>80</y>
           <width>761</width>
           <height>41</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QPushButton" name="startCleaning_pushButton">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>50</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>开始清洗</string>
         </property>
        </widget>
        <widget class="QPushButton" name="showCleaningInfo_pushButton">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>50</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>查看结果</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>字典</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>751</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="dict_stopWords_pushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="dict_stopWords_lineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>同义词</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>停用词</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="dict_userdict_pushButton">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>自定义</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="dict_userdict_lineEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="dict_tongyi_lineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="dict_tongyi_pushButton">
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>参数</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
